src/redux/Rockets.js

```diff
@@ -1,7 +1,28 @@
-import React from 'react';
+import axios from 'axios';
+import ROCKETS_API from '../Api/Api';
 
-const Rockets = () => (
-  <div />
-);
+const GET_ROCKETS = 'spaceApp/Rockets/GET_ROCKETS';
 
-export default Rockets;
+const initialRocketState = [];
+const RocketReducer = (state = initialRocketState, action) => {
+  switch (action.type) {
+    case GET_ROCKETS:
+      return [...state, action.rockets];
+
+    default:
+      return state;
+  }
+};
```

Comment:
Hi @Alfred-KInyua  please use the react-redux toolkit's methods createSlice() and createAynchThunk() to implement your project. it's part of the requirement.